cla-backend/serverless.yml

```diff
@@ -118,6 +118,7 @@ provider:
         - "arn:aws:dynamodb:#{AWS::Region}:#{AWS::AccountId}:table/cla-${opt:stage}-user-permissions"
         - "arn:aws:dynamodb:#{AWS::Region}:#{AWS::AccountId}:table/cla-${opt:stage}-users"
         - "arn:aws:dynamodb:#{AWS::Region}:#{AWS::AccountId}:table/cla-${opt:stage}-metrics"
+        - "arn:aws:dynamodb:#{AWS::Region}:#{AWS::AccountId}:table/cla-${opt:stage}-projects-cla-groups"
     - Effect: Allow
```

Comment:
We are missing the index permissions - if possible, would like to keep both the serverless.yml table and index configs the same.

Reply:
okay. Will update the v1 serverless config as well.